deparse.c

```diff
@@ -2425,8 +2445,30 @@ sqlite_deparse_direct_update_sql(StringInfo buf, PlannerInfo *root,
 			appendStringInfo(buf, "sqlite_fdw_uuid_str(");
 			special_affinity = true;
 		}
+		if (pg_attyp == TIMESTAMPOID && preferred_affinity == SQLITE_INTEGER)
+		{
+			appendStringInfo(buf, "strftime(");
+			special_affinity = true;
```

Comment:
Look like currently `timestamp` with `integer` affinity support was not full. Should I provide some TCs?

Reply:
Do you have any other Pull Request for `timestamp`? I think it is better to separate it to other Pull Request. This Pull Request is for MAC address only. It will be easier for reviewing and confirming.

Reply:
@son-phamngoc I have seen very big problems with mixed affinity `timestamp`. I decide here will be nothing about `timestamp`, please ensure no `timestamp` is touched in the PR. Maybe it will be PR about `timestamp`, but now I am not sure about this.